robot_ws/src/turtlebot_controller/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>turtlebot_controller</name>
+  <version>0.1.1</version>
+  <description>
+    Deployment package for the Object Tracker RoboMaker sample application on the Turtlebot3
+  </description>
+  
+  <license>Apache 2.0</license>
+  <author email="ros-contributions@amazon.com">AWS RoboMaker</author>
+  <maintainer email="ros-contributions@amazon.com">AWS RoboMaker</maintainer>
+
+  <depend>python-boto3-pip</depend>
```

Comment:
Does `Cannot locate rosdep definition for [python-boto3-pip]` happen for you?

Reply:
We checked by using **rosdep db | grep boto3**. It should be python3-boto3 instead of python-boto3-pip in package.xml.
Also, Verified changes using **rosdep install --from-paths src --ignore-src -r -y**